src/utils/winston.js

```diff
@@ -0,0 +1,19 @@
+// eslint-disable-next-line import/no-unresolved
+import { createLogger, format, transports } from 'winston';
+
+
+const { combine, timestamp, prettyPrint } = format;
+const logger = createLogger({
+  format: combine(
+    timestamp(),
+    prettyPrint(),
+  ),
+  transports: [
+    new transports.Console(),
+    new transports.File({ filename: '.logs/error.log', level: 'error' }),
+    new transports.File({ filename: './logs/info.log', level: 'info' }),
```

Comment:
Great job but I have tested it but the error log creates the  `.logs/error.log` which works okay but the info does not so I suggest maybe the line `./logs/info.log` to `.logs/info.log` for it to work.

Reply:
And also on the pr description how to manually  you should add the script to run so as to test it

Reply:
Oh yeah! Missed that. Fixed now. Thanks